libs/remix-ui/plugin-manager/src/types.d.ts

```diff
@@ -180,6 +180,7 @@ export interface FormStateProps {
   hash: string
   methods: any
```

Comment:
Almost everything looks good to me. Just it would be nice to avoid any as much as possible. methods seems to be string[] and canActivate a state dispatch. If we can be explicit with the types, it would be super! But everything else looks honky dory!